src/imio/esign/utils.py

```
@@ -153,17 +152,21 @@ def create_external_session(session_id, b64_cred=None, esign_root_url=None):
-    b64_cred = b64_cred or ESIGN_CREDENTIALS
-    if b64_cred:
-        headers["Authorization"] = "Basic {}".format(b64_cred)
+    headers = {
+        "accept": "application/json",
+        "Authorization": "Bearer %s" % get_auth_token(),
+    }
 
     logger.info(data_payload)
-    ret = post_request(
-        session_url, data={"data": json.dumps(data_payload, default=vars)}, headers=headers, files=files_payload
+    ret = requests.post(
```

Ruff (0.15.0)

[error] Probable use of `requests` call without timeout

(S113)